docs/django-widget.rst

```diff
@@ -10,23 +10,31 @@ Settings
 --------
 
 Besides required ``pub_key``, ``secret`` settings there are optional settings,
-for example, ``widget_version`` or ``widget_build``:
+for example, ``widget_version`` or ``widget_variant``:
 
 .. code-block:: python
 
     UPLOADCARE = {
         'pub_key': 'demopublickey',
         'secret': 'demoprivatekey',
-        'widget_version': '3.x',  // ~= 3.0 (latest)
-        'widget_build': 'min',  // without jQuery
+        'widget_version': '0.26.0',
+        'widget_variant': 'inline',  # regular | inline | minimal
         'cdn_base': 'https://cdn.mycompany.com',
     }
 
 PyUploadcare takes assets from Uploadcare CDN by default, e.g.:
 
 .. code-block:: html
 
-    <script src="https://ucarecdn.com/widget/x.y.z/uploadcare/uploadcare.full.min.js"></script>
+    <script
+        src="https://cdn.jsdelivr.net/npm/@uploadcare/blocks@0.26.0/web/lr-file-uploader-regular.min.js"
+        type="module"
+    ></script>
+    <lr-file-uploader-regular
+        css-src="https://cdn.jsdelivr.net/npm/@uploadcare/blocks@0.26.0/web/lr-file-uploader-regular.min.css"
+        ctx-name="..."
+    ></lr-file-uploader-regular>
```

Comment:
Can the widget version and variant in this example inherit one from lines 20-21?

Reply:
this is docs. probably not.
so we need wildcard verisons